exporters/auto/build.gradle

```diff
@@ -0,0 +1,20 @@
+plugins {
+    id "com.github.johnrengelman.shadow" version "6.0.0"
+}
+
+description = 'Auto Exporter for OpenTelemetry'
+
+dependencies {
+    repositories {
+        maven {
+            url "https://dl.bintray.com/open-telemetry/maven"
+        }
+    }
+
+    api(libraries.auto_value_annotations)
+    annotationProcessor(libraries.auto_value)
+    implementation("io.opentelemetry.instrumentation.auto:opentelemetry-javaagent-tooling:0.8.0")
```

Comment:
Could you please follow the existing pattern with library names and versions defined in https://github.com/GoogleCloudPlatform/opentelemetry-operations-java/blob/7cac7abd83372548da71a4a6fe463d90b167ffaf/build.gradle#L38-L39?

Most importantly, the `0.8.0` here should be replaced with https://github.com/GoogleCloudPlatform/opentelemetry-operations-java/blob/7cac7abd83372548da71a4a6fe463d90b167ffaf/build.gradle#L24 AFAIU (unless instrumentation repo has its own release cycles - then declare a separate constant).

Reply:
Yup, good call. This was me purely trying to get things working